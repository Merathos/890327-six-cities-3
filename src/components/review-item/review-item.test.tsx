import * as React from "react";
import * as renderer from "react-test-renderer";
import ReviewItem from "./review-item";

const review = {
  id: `51243`,
  name: `Max`,
  avatar: `img/avatar-max.jpg`,
  rating: 2.5,
  date: `2019-06-13T04:41:20`,
  text: `Consectetur sit velit nulla ipsum. Sunt occaecat amet cupidatat laboris commodo in. Culpa voluptate elit amet officia veniam cillum ad. Ea magna adipisicing laboris incididunt. Laborum eu sit consequat eu nisi. Nulla ipsum ea reprehenderit enim labore do et. Irure anim laborum aute quis.`
};


it(`Render Review Item`, () => {
  const tree = renderer
    .create(<ReviewItem
      key={review.id}
      review={review}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
