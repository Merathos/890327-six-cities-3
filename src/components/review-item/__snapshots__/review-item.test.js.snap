// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Review Item 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar-max.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Max
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "50%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Consectetur sit velit nulla ipsum. Sunt occaecat amet cupidatat laboris commodo in. Culpa voluptate elit amet officia veniam cillum ad. Ea magna adipisicing laboris incididunt. Laborum eu sit consequat eu nisi. Nulla ipsum ea reprehenderit enim labore do et. Irure anim laborum aute quis.
    </p>
    <time
      className="reviews__time"
      dateTime="2019-06-13"
    >
      June 2019
    </time>
  </div>
</li>
`;
